Tutorials/BoseHubbard1d/bosehubbard1d.py

```diff
@@ -12,67 +12,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import netket as nk
+from mpi4py import MPI
 
-from __future__ import print_function
-import json
+# 1D Periodic Lattice
+g = nk.graph.Hypercube(length=12, ndim=1, pbc=True)
 
-pars = {}
+# Boson Hilbert Space
+hi = nk.hilbert.Boson(graph=g, n_max=3, n_bosons=12)
 
-# defining the hilbert space
-pars['Hilbert'] = {
-    'Name': 'Boson',
-    'Nmax': 3,
-    'Nbosons': 12,
-}
+# Bose Hubbard Hamiltonian
+ha = nk.operator.BoseHubbard(U=4.0, hilbert=hi)
 
-# defining the lattice
-pars['Graph'] = {
-    'Name': 'Hypercube',
-    'L': 12,
-    'Dimension': 1,
-    'Pbc': True,
-}
+# Jastrow Machine with Symmetry
+ma = nk.machine.RbmSpinSymm(alpha=4, hilbert=hi)
 
-# defining the hamiltonian
-pars['Hamiltonian'] = {
-    'Name': 'BoseHubbard',
-    'U': 4.0,
-}
+# Sampler
+sa = nk.sampler.MetropolisHamiltonian(machine=ma, hamiltonian=ha)
 
-# defining the wave function
-pars['Machine'] = {
-    'Name': 'RbmSpinSymm',
-    'Alpha': 4.0,
-}
+# Stochastic gradient descent optimization
+op = nk.optimizer.AdaMax()
 
-# defining the sampler
-# here we use Metropolis sampling
-pars['Sampler'] = {
-    'Name': 'MetropolisHamiltonian',
-}
+# Variational Monte Carlo
+vmc = nk.gs.Vmc(
+    hamiltonian=ha,
+    sampler=sa,
+    optimizer=op,
+    nsamples=1000,
```

Comment:
`n_samples`

Reply:
nsamples is the currently supported name, but I can change this over too!